docs/ops/opset4.md

```diff
@@ -27,6 +27,7 @@ declared in `namespace opset4`.
 * [CTCGreedyDecoder](sequence/CTCGreedyDecoder_1.md)
 * [Ceiling](arithmetic/Ceiling_1.md)
 * [Clamp](activation/Clamp_1.md)
+* [Mish](activation/Mish_1.md)
```

Comment:
Change version to Mish_4

Reply:
The operations must be sorted according to name. Why "Mish" is located between "Clamp" and "Concat"?